dbms/tests/queries/0_stateless/00387_use_client_time_zone.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh -e
+
+TZ=UTC clickhouse --client --use_client_time_zone=1 --query="SELECT toDateTime(1000000000)"
```

Comment:
Most runs are from per commit builds in Jenkins. Packages are built, get installed and then tests are run.

For manual runs, I just copy compiled binary to /usr/bin/.